logging/zerolog/ctxzr/context.go

```diff
@@ -0,0 +1,80 @@
+package ctxzr
+
+import (
+	"context"
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	"github.com/rs/zerolog"
+)
+
+type ctxMarker struct{}
+
+type CtxLogger struct {
+	Logger *zerolog.Logger
+	Fields map[string]interface{}
+}
+
+var (
+	ctxMarkerKey = &ctxMarker{}
+	nullLogger   = &zerolog.Logger{}
+)
+
+func MergeFields(mp1 map[string]interface{}, mp2 map[string]interface{}) map[string]interface{} {
```

Comment:
Is there a way to do this without adding a exported func into this package ... it starts to move away from the other implementations if you do this.

Reply:
Lets at least move this to not being exported